### DISCUSSION TITLE
Is `useSWRInfinite` considered a stable API?

### DISCUSSION THREAD

**Discussion by yamgent** [vercel/swr#641] [General] [answered]
Hi! I would like to ask whether the `useSWRInfinite()` hook is considered a stable API (i.e. it can be used in production, and the API isn't going to change radically in the future).

Am wondering about this because there weren't any documentation about it in the README, so I wasn't sure whether it is still experimental, or the missing docs was just an oversight.

Thanks!

**Comment by HamzaAmar** [comment, marked as answer]
i dont know if the useSWRInfinite() is stable or not but if you want documentation you can find something that can help you here 

https://swr.vercel.app/docs/pagination#use-swr-infinite
https://swr.vercel.app/examples/infinite-loading

**Comment by yamgent** [comment]
Thanks for pointing out the links, my bad, I was looking at the README alone and didn't realise there's a full page on it already. 🤦‍♂️
